colorscad.sh

```diff
@@ -193,6 +198,11 @@ COLORS=$(
 	sed 's/^[^\[]*//'
 )
 
+if [ $FIXED_COLOR_ORDER -eq 1 ]; then
+  COLORS=$(echo "$COLORS" | sort -n)
```

Comment:
This is an important improvement indeed, but maybe we can have the best of both worlds.
If this sort is moved down just before the:
```echo "Generate a merged .${FORMAT} file"```
Then I bet the output will still have reproducible color order, while we can keep the highly optimistic parallelism-improving order here.

Would you mind trying if moving the sort down works for you?

It can probably be done unconditionally even, if it works then I'll have a look at updating all tests accordingly in the weekend.

Reply:
That's the best possible outcome.
It works just fine that way, I've removed the -r arg complete and just sort them as you recommended.

Thanks!

Reply:
Updated the test expectations, all of them are green again!